operator/config/rbac/nodeagent_editor_role.yaml

```diff
@@ -1,26 +1,25 @@
-# permissions for GameServer Pods (specifically the sidecar) to watch and patch GameServer CRs and GameServerDetail
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: gameserver-editor-role
-rules:
-- apiGroups:
-  - mps.playfab.com
-  resources:
-  - gameservers
-  verbs:
-  - list
-  - watch
-- apiGroups:
-  - mps.playfab.com
-  resources:
-  - gameservers/status
-  verbs:
-  - patch
-- apiGroups:
-  - mps.playfab.com
-  resources:
-  - gameserverdetails
-  verbs:
-  - get
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Nit: I like the comment at the top of the old file describing what the intent is